crates/deltalake-core/src/operations/mod.rs

```diff
@@ -14,6 +14,8 @@ use crate::errors::{DeltaResult, DeltaTableError};
 use crate::table::builder::DeltaTableBuilder;
 use crate::DeltaTable;
 
+#[cfg(feature = "parquet")]
```

Comment:
i think we also need to enable the `arrow` feature.

Reply:
Added the `arrow` feature.